source/_integrations/binary_sensor.rest.markdown

```diff
@@ -66,17 +66,29 @@ sensor:
 {% endraw %}
 
 {% configuration %}
-resource:
-  description: The resource or endpoint that contains the value.
-  required: true
+authentication:
```

Comment:
These are available via the integation configuration, however, you are editing the platform configuration? (please note, we are trying to get rid of platform configurations, so even if it was possible I don't think we should advertise this).

Reply:
I see. These were "synced" from the [RESTful Sensor](https://www.home-assistant.io/integrations/sensor.rest/) documentation. If they should not be listed here for RESTful Binary Sensor, then they should - for the sake of consistency - also be removed from the RESTful Sensor?